src/com/codeenginestudio/management/book/dao/BookTypeDao.java

```diff
@@ -0,0 +1,80 @@
+package com.codeenginestudio.management.book.dao;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+
+import com.codeenginestudio.management.book.model.BookAndBookType;
+import com.codeenginestudio.management.book.model.BookType;
+import com.codeenginestudio.management.book.util.HibernateUtil;
+
+public class BookTypeDao {
+
+	private BookAndBookTypeDao bookAndBookTypeDao = new BookAndBookTypeDao();
+
+	@SuppressWarnings("unchecked")
+	public List<BookType> getBookTypes() {
+
+		Transaction transaction = null;
+		List<BookType> listOfBookType = null;
+
+		try (Session session = HibernateUtil._getSessionFactory().openSession()) {
+
+			transaction = session.beginTransaction();
+			listOfBookType = session.createQuery("from BookType").getResultList();
+			transaction.commit();
+
+		} catch (Exception e) {
+
+			if (transaction != null) {
+
+				transaction.rollback();
+			}
+
+			System.out.println("getBookTypes() failed: " + e.getMessage());
```

Comment:
@thuycao625 @HuongNguyenThiThu Should use log library like log4j, not `system.out`